src/constructs/loadbalancing/alb.test.ts

```diff
@@ -171,13 +145,12 @@ describe("The GuApplicationListener class", () => {
 
     new GuApplicationListener(stack, "ApplicationListener", {
       loadBalancer,
-      overrideId: true,
+      existingLogicalId: "AppListener",
       defaultAction: ListenerAction.forward([targetGroup]),
       certificates: [{ certificateArn: "" }],
     });
 
-    const json = SynthUtils.toCloudFormation(stack) as SynthedStack;
-    expect(Object.keys(json.Resources)).toContain("ApplicationListener");
+    expect(stack).toHaveResourceOfTypeAndLogicalId("AWS::ElasticLoadBalancingV2::Listener", "AppListener");
   });
 
   test("does not override the id if the prop is false", () => {
```

Comment:
Minor, should this now be something like:

`does not override the id if the existingLogicalId prop is not defined`

Reply:
Actually, I wonder again if this test is really adding any value given that this is now controlled outside of this construct?